src/index.css

```diff
@@ -0,0 +1,5 @@
+body {
```

Comment:
[OPTIONAL] Good job with the styling, it looks nice 👍  Since we need to follow design guideline, I think it would be nice if you use Lato and Gill Sans. 👍